include/analog.h

```diff
@@ -1,6 +1,13 @@
 #ifndef __ANALOG_H__
 #define __ANALOG_H__
 
+#include <Arduino.h>
+#include "adc.h"
+#include "ina281.h"
```

Comment:
I don't think the ina281 is used in `analog.cpp`. Also, the PDC uses the INA180A1QDBVRQ1.